src/Wallabag/CoreBundle/Controller/WallabagRestController.php

```diff
@@ -72,7 +74,10 @@ public function getEntriesAction(Request $request)
             throw $this->createNotFoundException();
         }
 
-        return $entries;
+        $hateoas = HateoasBuilder::create()->build();
+        $json = $hateoas->serialize($entries, 'json');
```

Comment:
Regarding the bundle documentation, you don't need to call HateoasBuilder manually, you can do that:

```php
$json = $this->get('serializer')->serialize($entries, 'json');
```